lib/Doctrine/DBAL/Types/DateTimeImmutableType.php

```diff
@@ -13,14 +13,18 @@ class DateTimeImmutableType extends DateTimeType
 {
     /**
      * {@inheritdoc}
+     *
+     * @return string
```

Comment:
To be dropped.

Reply:
Shouldn't PHPCS detect this as redundant?